data_safe_haven/console/pretty.py

```diff
@@ -0,0 +1,42 @@
+from typing import Any
+
+from rich.console import Console, JustifyMethod, OverflowMethod
+from rich.style import Style
+
+console = Console()
+
+
+def pretty_print(
+    *objects: Any,
+    sep: str = " ",
+    end: str = "\n",
+    style: str | Style | None = None,
+    justify: JustifyMethod | None = None,
+    overflow: OverflowMethod | None = None,
+    no_wrap: bool | None = None,
+    emoji: bool | None = None,
+    markup: bool | None = None,
+    highlight: bool | None = None,
+    width: int | None = None,
+    height: int | None = None,
+    crop: bool = True,
+    soft_wrap: bool | None = None,
+    new_line_start: bool = False,
```

Comment:
Do we actually want to use any of these options in our code? If not, why are we exposing them as arguments to `pretty_print`?